docker/terraform/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hashicorp/terraform:0.12.24
+FROM hashicorp/terraform:0.12.26
```

Comment:
please use the gfw terraform image

https://github.com/wri/gfw-data-api/blob/master/terraform/docker/docker-compose.yml